src/Components/Admin/index.js

```diff
@@ -0,0 +1,138 @@
+import React, {Component} from 'react';
+import { thisExpression } from '@babel/types';
+
+class Admin extends Component {
+    check(e){
+        console.log(e.target);
+    }
+    state = {
+        events: [
+            {
+                name: "name1",
+                etime: {
+                    hrs: 2,
+                    min: 1,
+                    day: 3
+                },
+                venue: "venue 1",
+                winner: "asdf sadf",
+                announcements: "announcement 1\nannouncement 2\nannouncement 3"
+            }
+        ],
+        targetEvent: {
+            name: "enter name",
+            etime: {
+                hrs: 2,
+                min: 1,
+                day: 2
+            },
+            venue: "",
+            winner: "",
+            announcements: ""
+        },
+        yetToSelectEvent: true,
+        targetName: ""
+    }
+
+    // make copy of entire struct, ref: https://stackoverflow.com/questions/43638938/updating-an-object-with-setstate-in-react
+
```

Comment:
I wanted to keep `targetEvent` as a struct to avoid complications, so I had to manually copy.